### ISSUE TITLE
no user feedback when saving project setting page for the second time

### ISSUE THREAD

**Issue by ntraut** [neuroanatomy/BrainBox#374]
## Thank you for your feedback.

Is it a [x] bug report or a [ ] feature request?

### Current behaviour

I think we already discussed the issue even if I cannot recall where. When we save project settings for the first time, the message "Project settings updated." is displayed. This message never disappears so when we save for the second time we don't see any feedback.

### Expected behaviour

Something new should happen when we save for the second time. For example the updated message could fade out after some time so we can distinguish a new message from an old one.

**Comment by katjaq** [comment]
Yes! Thank you! I remember we discussed the issue. For me, expected behaviour would be that the first message fades away after 5 seconds or so. That makes it consistent: every time you save, you get this message for a few seconds.
